cli/cmd/transformCmd.go

```diff
@@ -86,22 +89,42 @@ func doTransform() error {
 		if err != nil {
 			return "", err
 		}
+
+		s := string(b)
+		if ndjson {
+			return s, nil
+		}
+
 		return strings.Join(
 			strs.NoErrMapSlice(
-				strings.Split(jsons.BPJ(string(b)), "\n"),
+				strings.Split(jsons.BPJ(s), "\n"),
 				func(s string) string { return "\t" + s }),
 			"\n"), nil
 	}
 
 	record, err := doOne()
 	if err == io.EOF {
-		fmt.Println("[]")
+		if ndjson {
+			fmt.Println("")
+		} else {
+			fmt.Println("[]")
+		}
```

Comment:
well this is just written weirdly,  why not something inline with:
```
if !stream {
    println("[]")
}
```

Reply:
Println writes a new line even with a blank string, should we just leave it blank?